Time/DayAndNightClock

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# <bitbar.title>DayAndNightClock</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>FusionX</bitbar.author>
+# <bitbar.author.github>FusionX13</bitbar.author.github>
+# <bitbar.desc>A clock that can do many things.</bitbar.desc>
+# <bitbar.image>https://ibb.co/CBLByKf</bitbar.image>
+# <bitbar.dependencies>bash</bitbar.dependencies>
+# <bitbar.abouturl>http://url-to-about.com/</bitbar.abouturl>
+
+# Gets Emoji Moon
+get_phase_day () {
+  local lp=2551443
+  local now=$(date -ju +"%s")
+  local newmoon=592500
+  local phase=$((($now - $newmoon) % $lp))
+  echo $(((phase / 86400) + 1))
+}
+
+# Gets Emoji Moon
+get_moon_icon () {
+  local phase_number=$(get_phase_day)
```

Comment:
```
n /Users/bhartvig/Projects/bitbar-plugins/Time/DayAndNightClock.1h.sh line 23:
  local phase_number=$(get_phase_day)
        ^----------^ SC2155: Declare and assign separately to avoid masking return values.
```